### ISSUE TITLE
p-inputNumber Accessibility Issues

### ISSUE THREAD

**Issue by aciesla4** [primefaces/primeng#18814]
### Describe the bug

The p-inputNumber component has two accessibility violation when running the IBM Equal Accessibility Checker: 
1. The ARIA attribute "aria-valuemin" is in conflict with the corresponding HTML attribute "min"
2. The ARIA attribute "aria-valuemax" is in conflict with the corresponding HTML attribute "max"

### Reason for not contributing a PR

- [x] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://tcthglgb.stackblitz.io 

### Environment

Stackblitz

### Angular version

20.0.4

### PrimeNG version

v20

### Steps to reproduce the behavior

1. Install the [IBM Equal Accessibility Checker](https://chromewebstore.google.com/detail/ibm-equal-access-accessib/lkcagbfjnkomcinoddgooolagloogehp)
2. Open the stackblitz
3. Run the accessibility checker
4. Filter on violations only and note the errors: 'The ARIA attribute "aria-valuemin" is in conflict with the corresponding HTML attribute "min"' and 'The ARIA attribute "aria-valuemax" is in conflict with the corresponding HTML attribute "max"'

### Expected behavior

No accessibility violations are present for the p-inputNumber component
